src/pylibsshext/channel.pyx

```diff
@@ -170,9 +180,23 @@ cdef class Channel:
 
         return result
 
+    def send_eof(self):
+        rc = libssh.ssh_channel_send_eof(self._libssh_channel)
+        if rc != libssh.SSH_OK:
+            raise LibsshChannelException("Failed to ssh_channel_send_eof: [%d]" % rc)
+
+    def send_signal(self, sig):
```

Comment:
Could you make this accept constants from the standard library module called `signal`?
Also, add a docstring, please.